virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc

```diff
@@ -20,6 +22,8 @@ include::modules/virt-creating-a-vm-from-a-template-with-an-attached-boot-source
 
 include::modules/virt-creating-a-custom-image-boot-source-web.adoc[leveloffset=+1]
 
+A Quick Start tour is available in the {product-title} web console to assist you in uploading a boot source. 
```

Comment:
I would move this to the top of the assembly.

Reply:
Done.